### DISCUSSION TITLE
Install rahat-agency to your environment and send us your feedback.

### DISCUSSION THREAD

**Discussion by nishu-rumsan** [esatya/rahat-agency#233] [Feedbacks]
Install [rahat](https://github.com/esatya/rahat-agency) to your environment by following the sets listed in [Readme](https://github.com/esatya/rahat-agency#readme).

When installing rahat-agency in your environment, how easy was the process? 

Share your feedback and suggestions about the installation process [here](https://github.com/esatya/rahat-agency/discussions/233).

**Comment by saminarp** [comment]
Hello, 
I am using MacOS and I tried installing it in my environment. Initially I got this error after running `yarn install` 
![image](https://user-images.githubusercontent.com/71540402/196648202-20cd9f11-70cf-4e51-8e3c-895a716aaa96.png)

I was suggested to remove the `rumsun-ui` package and try again. 

However, the subsequent error looked like this. 

<img width="1523" alt="CleanShot 2022-10-19 at 05 04 06@2x" src="https://user-images.githubusercontent.com/71540402/196648797-f19bc9d0-9027-453e-8fe8-6468b63b853a.png">

Please advise what I should do as I would really love to contribute to this amazing project!

**Comment by shahgnp** [comment]
Hello again,
the link in the last line is broken here too

**Reply by nishu-rumsan** [reply]
> Hello again, the link in the last line is broken here too

Updated.
